### PR TITLE
Add json endpoint doc

### PR DESCRIPTION
**Pull request by cob16**

### CODE REVIEW

File: documentation/submitting/submitting.md [hunk 1]
```diff
@@ -34,6 +33,17 @@ In order to send the data to the service team, there must be an email address to
 - JSON (TODO)
 - CSV (TODO)
 
+## JSON output
+
+Form submitter can send a webhook to your backend service to process submissions automatically.
+A representation of the users' answers is in JSON format data to a chosen destination.
+This JSON data is encrypted using a shared key via the [JWE standard](https://tools.ietf.org/html/rfc7516).
```
Comment (asmega) [inline documentation/submitting/submitting.md]: might be good to add must be exactly 16 characters long with our current implementation

File: documentation/submitting/submitting.md [hunk 2]
```diff
@@ -34,6 +33,17 @@ In order to send the data to the service team, there must be an email address to
...
+
+An example of custom integration is the  application cis the [hmcts-complaints-formbuilder-adapter](https://github.com/ministryofjustice/hmcts-complaints-formbuilder-adapter)
```
Comment (asmega) [inline documentation/submitting/submitting.md]: double spacing and `cis/is`

File: documentation/submitting/submitting.md [hunk 3]
```diff
@@ -34,6 +33,17 @@ In order to send the data to the service team, there must be an email address to
...
+
+Details to integrate with a custom endpoint
+- The Destination URL of the JSON payload is a Publisher config parameter  `SERVICE_OUTPUT_JSON_ENDPOINT.`
```
Comment (asmega) [inline documentation/submitting/submitting.md]: double spacing

**Comment by tommotaylor** [comment]
Closed in favour of https://github.com/ministryofjustice/form-builder/pull/38

**Pull request closed without merging**